notebooks/official/custom/sdk-custom-image-classification-batch.ipynb

```diff
@@ -128,182 +131,124 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "install_aip"
+        "id": "3b1ffd5ab768"
       },
       "source": [
-        "### Install additional packages\n",
-        "\n",
-        "Install the following packages for executing this notebook."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "id": "1fd00fa70a2a"
-      },
-      "outputs": [],
-      "source": [
-        "# Install the packages\n",
-        "! pip3 install --upgrade google-cloud-aiplatform \\\n",
-        "                        google-cloud-storage \\\n",
-        "                        pillow  \\\n",
-        "                        numpy    "
+        "## Getting Started"
```

Comment:
Get started 

- no gerunds in headers
- lower case for "started"

Reply:
Did required changes